src/list/_list.scss

```diff
@@ -0,0 +1,90 @@
+@import "../variables";
+@import "../mixins";
+
+
+.mdl-list {
+  display: block;
+  padding: 8px 0 ;
+  font-family: $preferred_font;
+  list-style: none;
+}
+
+
+.mdl-list__item {
+  @include typo-body-1();
+  display: flex;
+  box-sizing: border-box;
+  height: 48px;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: center;
+  padding : 16px 16px 20px 16px;
+  cursor: default;
+
+  & .mdl-list__item-sub-header {
+    padding: 0 0 0 16px;
+  }
+
+  & .mdl-list__item-primary-content {
+    order:0;
+    flex-grow: 2;
+
+    & a {
+      @include typo-body-1();
+      color: $default-item-text-color;
+      text-decoration: none;
+    }
+
+    & .material-icons {
+      font-size: 40px;
```

Comment:
The reason for doing this was to keep the spacing simple while being spec compliant.  A list item needs 72px padding left (when used with icon or avatar). 
So 16px + 40px (from icon) + 16px (from list item text) = 72px.  

I'm not to sure about this either, as mentioned above. what do you think?